### PR TITLE
Add OSLS summary blog post

### PR DESCRIPTION
**Pull request by caniszczyk**
List over some of the presentations from OSLS!
http://events.linuxfoundation.org/events/open-source-leadership-summit

Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

**Comment by caniszczyk** [comment]
RFC @gyehuda @jeffmcaffer anyone else :)

Would just love another set of eyes on it before publishing tomorrow

File: _posts/2017-03-21-todo-osls.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "TODO at Open Source Leadership Summit"
+author: todogroup
+---
+
+Last month, the TODO Group led a track at the [Open Source Leadership Summit (OSLS)](http://events.linuxfoundation.org/events/open-source-leadership-summit)
+which featured a variety of talks dedicated to open source program management:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">via our track at <a href="https://twitter.com/hashtag/lfosls?src=hash">#lfosls</a>: &quot;Open source @ scale: 10,000 engineers and counting&quot; <a href="https://t.co/tG0BFJpCZj">https://t.co/tG0BFJpCZj</a> <a href="https://t.co/sGJYeWVsSm">pic.twitter.com/sGJYeWVsSm</a></p>&mdash; TODO Group (@todogroup) <a href="https://twitter.com/todogroup/status/835172058166349824">February 24, 2017</a></blockquote>
```
Comment (jeffmcaffer) [inline _posts/2017-03-21-todo-osls.md]: fine to quote this but that is just one of the talks.  Perhaps it is just the GitHub MD preview but it looks a little funny and out of context.

Comment (gyehuda) [inline _posts/2017-03-21-todo-osls.md]: Nithya and I gave a talk too http://sched.co/9Khe I'm not sure why it was not classified under the todo track, but it certainly was in line with the rest.

Comment (gyehuda) [inline _posts/2017-03-21-todo-osls.md]: Note: we kinda retitled it to "M&A Deal Diligence and its Open Source Benefits"

**Comment by caniszczyk** [comment]
posted blog here: http://todogroup.org/blog/todo-osls/

**Pull request merged**